msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -5,16 +5,16 @@ ms.topic: reference
 ms.localizationpriority: medium
 ms.date: 11/15/2021
 ---
-# Public developer preview manifest schema for Teams
+# Public developer preview app manifest schema
 
 For information on how to enable developer preview, see [public developer preview for Microsoft Teams](~/resources/dev-preview/developer-preview-intro.md).
 
 > [!NOTE]
-> If you aren't using developer preview features, including running [Teams personal tabs and message extensions in Outlook and Microsoft 365 app](../../m365-apps/overview.md), use the [app manifest for GA features](~/resources/schema/manifest-schema.md) instead.
+> If you aren't using developer preview features, including running [Teams personal tabs and message extensions in Outlook and Microsoft 365 app](../../m365-apps/overview.md), use the [app manifest(previously called Teams app manifest)  for GA features](~/resources/schema/manifest-schema.md) instead.
```

Comment:
This link text doesn't reflect the updated name of the linked article:

[app manifest(previously called Teams app manifest)  for GA features](~/resources/schema/manifest-schema.md)

. . . could we change it to:

[app manifest](~/resources/schema/manifest-schema.md) (previously called Teams app manifest) for generally available features

Reply:
Updated.